### PR TITLE
Added benchmarks

### PR DESCRIPTION
**Pull request by dantleech**
Adds benchmarks see also: https://github.com/jackalope/jackalope-doctrine-dbal/pull/274

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "require-dev": {
         "psr/log": "~1.0",
         "phpcr/phpcr-api-tests": "2.1.4",
-        "symfony/console": "~2.0"
```
Comment (dantleech) [inline composer.json]: I couldn't see how this was a dev requirement ?

File: composer.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
...
+        "phpcr/phpcr-benchmarks": "~1.0",
+        "phpunit/phpunit": "4.7.*"
```
Comment (dantleech) [inline composer.json]: The benchmark suite depends on the PHPUnit bootstraping, which apparently needs some phpunit classes..

Comment (lsmith77) [inline composer.json]: shouldn't this then be a phpcr-benchmarks dependency then?

File: composer.json (hunk 1 above)
Comment (dbu) [inline composer.json]: we provide a command to start the jackrabbit.jar. which is probably not the best idea in the first place. and there is currently no test for it, so we can just as well drop this here.

Comment (dbu) [inline composer.json]: should be in suggests however, saying that you then can use commands from jackalope/jackalope and this repo...

**Comment by lsmith77** [comment]
@dantleech should we warm this topic up again? I see you have gone through a few refactorings if your benchmark suite ..

**Pull request closed without merging**